monkey/infection_monkey/monkey.py

```diff
@@ -116,9 +114,6 @@ def start(self):
             monkey_tunnel.start()
 
         StateTelem(False).send()
-
-        self._default_server = WormConfiguration.current_server
```

Comment:
My only concern is that I moved this code to execute earlier and overlooked code that will change `WormConfiguration.current_server`, but it doesn't seem so.